### PR TITLE
update `createTestDBObjects` unit test to use Tracker geometry T33

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

This PR is a companion of https://github.com/cms-sw/cmssw/pull/45764.
In response to https://github.com/cms-sw/cmssw/pull/45370#issuecomment-2301200925, this PR is meant to update the unit tests in `CondTools/SiPixel` to use the Tracker geometry T33 as the previously used geometry T21 is not available anymore. 

#### PR validation:

`scram b runtests_createDBObjects` runs fine.
N.B.: this PR requires an update of the external `cms-data` repository: https://github.com/cms-data/CalibTracker-SiPixelESProducers/pull/4

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by mmusich** [comment]
test parameters:
   * pull_request = https://github.com/cms-data/CalibTracker-SiPixelESProducers/pull/4

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
type bug-fix

**Comment by mmusich** [comment]
urgent 
   * fixes IB failures

**Comment by mmusich** [comment]
@cmsbuild ignore tests-rejected with ib-failure

**Comment by mmusich** [comment]
Unit test runs correctly now: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d51a17/41083/unitTests/src/CondTools/SiPixel/test/createDBObjects/testing.log

File: CondTools/SiPixel/test/createTestDBObjects.sh [hunk 1]
```diff
@@ -11,16 +11,16 @@ echo -e "TESTING Pixel 1D GenErr DB code ..."
...
-cmsRun ${SCRAM_TEST_PATH}/SiPixelGenErrorDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT615 Map=${SCRAM_TEST_PATH}/../data/phase2_T21_mapping.csv geometry=T21 GenErrFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v6.1.5_25x100_irradiated_v2_mc || die "Failure running SiPixelGenErrorDBObjectUploader_Phase2_cfg.py" $?
+cmsRun ${SCRAM_TEST_PATH}/SiPixelGenErrorDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT711 Map=${SCRAM_TEST_PATH}/../data/phase2_T33_mapping.csv geometry=T33 GenErrFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v7.1.1_25x100_v1_mc || die "Failure running SiPixelGenErrorDBObjectUploader_Phase2_cfg.py" $?
 
-echo -e "TESTING Pixel 2D Template DB code for Phase-2 ..."
-cmsRun  ${SCRAM_TEST_PATH}/SiPixel2DTemplateDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=1 Append=mc_25x100_IT615 Map=${SCRAM_TEST_PATH}/../data/phase2_T21_mapping_den.csv TemplateFilePath=CalibTracker/SiPixelESProducers/data/Phase2_IT_v6.1.5_25x100_irradiated_v2_mc denominator=True || die "Failure running SiPixel2DTemplateDBObjectUploader_Phase2_cfg.py" $?
+#echo -e "TESTING Pixel 2D Template DB code for Phase-2 ..."
```
Comment (perrotta) [inline CondTools/SiPixel/test/createTestDBObjects.sh:19]: Isn't this test portable to T33?

Comment (mmusich) [reply CondTools/SiPixel/test/createTestDBObjects.sh:19]: No

**Comment by perrotta** [comment]
+db

File: CondTools/SiPixel/test/createTestDBObjects.sh (hunk 1 above)
Comment (mmusich) [reply CondTools/SiPixel/test/createTestDBObjects.sh:19]: to elaborate further, the 2D templates are used in the IT digitizer to simulate radiation damage in the planar sensors (see https://github.com/cms-sw/cmssw/pull/40349 and https://github.com/cms-sw/cmssw/pull/40417). 
According to @tvami there is no 2D template payload available for the T33  (and above) tracker geometries, so apparently there weren't any irradiation scenarios considered (yet). 
I am keeping the line commented to re-instantiate the test when  at a later time an irradiation scenario becomes available for T33 (or for some subsequent - even final - tracker geometry)

**Comment by mandrenguyen** [comment]
+1
merging #45775 and #45764 together

**Comment by mmusich** [comment]
@mandrenguyen, thanks for merging.
This one requires also a `cms-data` update as well: https://github.com/cms-data/CalibTracker-SiPixelESProducers/pull/4

**Pull request merged**